internal/apiserver/server.go

```diff
@@ -347,6 +355,7 @@ func (as *apiServer) createMuxRouter(ctx context.Context, mgr namespace.Manager)
 
 	r.HandleFunc(`/api/swagger{ext:\.yaml|\.json|}`, hf.APIWrapper(as.swaggerHandler(as.swaggerGenerator(routes, apiBaseURL))))
 	r.HandleFunc(`/api`, hf.APIWrapper(hf.SwaggerUIHandler(publicURL+"/api/swagger.yaml")))
+	r.HandleFunc(`/api/version`, hf.APIWrapper(as.versionInfo(mgr)))
```

Comment:
I'm debating whether this should be a "normal" route that shows up in `routes.go` and lives under `/api/v1/version`. But maybe it's "global" enough to be outside the versioning of the FireFly API itself.